lib/backend/etcdbk/etcd.go

```diff
@@ -315,6 +304,49 @@ func New(ctx context.Context, params backend.Params, opts ...Option) (*EtcdBacke
 	return b, nil
 }
 
+func (b *EtcdBackend) checkVersion(ctx context.Context) error {
+	ctx, cancel := context.WithTimeout(ctx, time.Second*9)
+	results := make(chan error, len(b.cfg.Nodes))
+
+	var wg sync.WaitGroup
+	for _, nn := range b.cfg.Nodes {
+		wg.Add(1)
+		go func(n string) {
+			defer wg.Done()
+			status, err := b.clients.Next().Status(ctx, n)
+			if err != nil {
+				results <- trace.Wrap(err)
+				return
+			}
+
+			ver := semver.New(status.Version)
+			min := semver.New(teleport.MinimumEtcdVersion)
```

Comment:
`semver.New` will panic if we fail to parse the version. Should we return the error instead?